indexer/src/main/java/org/pdxfinder/dataloaders/updog/domainobjectcreation/DomainObjectCreator.java

```diff
@@ -1051,6 +1051,8 @@ public String getCellAsText(Row row, String columnName){
                 return Double.toString(row.getDouble(columnName));
             } else if (row.getColumnType(columnName) == ColumnType.INTEGER) {
                 return Integer.toString(row.getInt(columnName));
+            } else if (row.getColumnType(columnName) == ColumnType.LOCAL_DATE) {
+                return row.getDate(columnName).toString();
```

Comment:
This method looks nice a good candidate for a switch statement, given the number of execution branches. It even seems to give some small performance benefits! (https://stackoverflow.com/a/767849/5440828)

Reply:
Great idea!

Reply:
boxed primitive, primitives or enums are only allowed in switch statments.